app/controllers/alchemy/admin/essence_pictures_controller.rb

```diff
@@ -19,7 +19,7 @@ def edit
       def crop
         if @picture = @essence_picture.picture
           @content = @essence_picture.content
-          options_from_params[:format] ||= (configuration(:image_store_format) || 'png')
+          options_from_params[:format] ||= configuration(:image_store_format) || @picture.image_file_format || 'png'
```

Comment:
Like that change. We want to be sure that animated gifs don't animate during cropping, though. We should pass `flatten: true` as well then.

Reply:
Also tests are mandatory. Please add some.